engine/apps/api/serializers/channel_filter.py

```diff
@@ -90,6 +91,10 @@ def validate_filtering_term(self, filtering_term):
                 raise serializers.ValidationError(["Filtering term is incorrect"])
         return filtering_term
 
+    def validate_filtering_term_jinja2(self, filtering_term_jinja2):
+        # TODO: check if valid
+        return filtering_term_jinja2
```

Comment:
`apply_jinja_template` function from `engine/common/jinja_templater` might be useful here